frontend/ui/size.lua

```diff
@@ -53,6 +53,7 @@ local Size = {
         button = Screen:scaleBySize(2),
         buttontable = Screen:scaleBySize(4),
         fullscreen = Screen:scaleBySize(15),
+        fullscreen_popout = Screen:scaleBySize(3), -- Size.border.window * 2
```

Comment:
Should probably be in Size.margin rather than Size.padding, cf https://github.com/koreader/koreader/pull/7335#discussion_r582081215